### DISCUSSION TITLE
How are images resized

### DISCUSSION THREAD

**Discussion by an3252sv** [AlexeyAB/darknet#8428] [Q&A] [answered]
I have a data set with 1280x720 sized images. If I set the network parameters width=416 and height=416, how will my images be resized? I have read that the images aspect ratio will be preserved i.e, they get downscale to 416x234 (same aspect ratio as 1280x720), then padding is added to make the image 416x416. Source: https://github.com/pjreddie/darknet/issues/800, written by 'pullmyleg' on Apr 6 2020. Their answer refers to yolov3, but is the same method used for yolov4? If that is the case, I would really like to be pointed to the resource where this is stated.

**Comment by stephanecharette** [comment, marked as answer]
Aspect ratio is not preserved:  https://www.ccoderun.ca/programming/darknet_faq/#square_network
Information on image dimensions vs network dimensions:  https://www.ccoderun.ca/darkmark/ImageSize.html
